### PR TITLE
TTZJetsTobb Cards

### PR DESCRIPTION
**Pull request by wweiphy**

### CODE REVIEW

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/TTZJetsTobb_5f_NLO/TTZJetsTobb_5f_NLO_madspin_card.dat [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+#initialization parameters
+ set Nevents_for_max_weigth 250 # number of events for the estimate of the max. weight
+ set max_weight_ps_point 400  # number of PS to estimate the maximum for each event
+# 
+
+set max_running_process 1
+
+define q = b
+define q~ = b~
```
Comment (mlizzo) [inline bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/TTZJetsTobb_5f_NLO/TTZJetsTobb_5f_NLO_madspin_card.dat:24]: These lines seem useless to me, just keep the b quark

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/TTZJetsTobb_5f_NLO/TTZJetsTobb_5f_NLO_madspin_card.dat [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+# specify the decay for the final state particles
+#(inclusive top decays in this case)
+decay t > w+ b, w+ > all all
+decay t~ > w- b~, w- > all all
+decay z > q q~
```
Comment (mlizzo) [inline bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/TTZJetsTobb_5f_NLO/TTZJetsTobb_5f_NLO_madspin_card.dat:30]: change q to b

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/TTZJetsTobb_5f_NLO/TTZJetsTobb_5f_NLO_madspin_card.dat [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+set max_running_process 1
+
+define q = b
+define q~ = b~
+
+# specify the decay for the final state particles
+#(inclusive top decays in this case)
+decay t > w+ b, w+ > all all
+decay t~ > w- b~, w- > all all
```
Comment (mlizzo) [inline bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/TTZJetsTobb_5f_NLO/TTZJetsTobb_5f_NLO_madspin_card.dat:29]: Just to be sure, are you interested in inclusive ttbar decays?

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/TTZJetsTobb_5f_NLO/TTZJetsTobb_5f_NLO_run_card.dat [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+#***********************************************************************
+    1   = lpp1    ! beam 1 type (0 = no PDF)
+    1   = lpp2    ! beam 2 type (0 = no PDF)
+ 6500   = ebeam1  ! beam 1 energy in GeV
+ 6500   = ebeam2  ! beam 2 energy in GeV
+#***********************************************************************
+# PDF choice: this automatically fixes also alpha_s(MZ) and its evol.  *
+#***********************************************************************
+ lhapdf    = pdlabel   ! PDF set                                     
+$DEFAULT_PDF_SETS = lhaid     ! if pdlabel=lhapdf, this is the lhapdf number
```
Comment (mlizzo) [inline bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/TTZJetsTobb_5f_NLO/TTZJetsTobb_5f_NLO_run_card.dat:58]: please also add the following lines:
 $DEFAULT_PDF_SETS = lhaid
 $DEFAULT_PDF_MEMBERS  = reweight_PDF

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/TTZJetsTobb_5f_NLO/TTZJetsTobb_5f_NLO_run_card.dat [hunk 5]
```diff
@@ -0,0 +1,146 @@
...
+# Reweight flags to get scale dependence and PDF uncertainty           *
+# For scale dependence: factor rw_scale_up/down around central scale   *
+# For PDF uncertainty: use LHAPDF with supported set                   *
+#***********************************************************************
+ .true.   = reweight_scale   ! reweight to get scale dependence
+  0.5     = rw_Rscale_down   ! lower bound for ren scale variations
+  2.0     = rw_Rscale_up     ! upper bound for ren scale variations
+  0.5     = rw_Fscale_down   ! lower bound for fact scale variations
+  2.0     = rw_Fscale_up     ! upper bound for fact scale variations
+$DEFAULT_PDF_MEMBERS = reweight_PDF     ! reweight to get PDF uncertainty
```
Comment (mlizzo) [inline bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/TTZJetsTobb_5f_NLO/TTZJetsTobb_5f_NLO_run_card.dat:95]: I would move this line below line 58, for the sake of readability

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/TTZJetsTobb_5f_NLO/TTZJetsTobb_5f_NLO_customizecards.dat [hunk 6]
```diff
@@ -0,0 +1,4 @@
+#put card customizations here (change top and higgs mass for example)
+set param_card mass 6 172.5
+set param_card yukawa 6 172.5
+set param_card mass 25 125.0
```
[the remark this answers is not in the document]
Comment (wweiphy) [reply bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/TTZJetsTobb_5f_NLO/TTZJetsTobb_5f_NLO_customizecards.dat:4]: You are right. I'll remove this card.

**Comment by mlizzo** [comment]
I have no further comments, please @menglu21 @Saptaparna @GurpreetSinghChahal have a look at it

**Pull request merged**